pkg/pushsync/pushsync_test.go

```diff
@@ -488,7 +488,7 @@ func TestPushChunkToNextClosest(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if ta2.Get(tags.StateSent) != 2 {
```

Comment:
this seems a bit wrong no? why is this change needed?

Reply:
restored as well